README.md

```diff
@@ -237,7 +237,7 @@ To use Kedro-Viz as a standalone React component, you can follow the example bel
 import KedroViz from '@quantumblack/kedro-viz';
 import '@quantumblack/kedro-viz/lib/styles/styles.min.css';
 
-const MyApp = () => <KedroViz data={json} />;
+const MyApp = () => <div style={{height: `100vh`}}><KedroViz data={json} /></div>;
```

Comment:
can we add options here. thanks

Reply:
@rashidakanchwala like this?
```
<div style={{height: `100vh`}}>
  <KedroViz
      data={json}
      options={
        display: {
          globalToolbar: true,
          sidebar: true,
          metadataPanel: true,
          miniMap: true,
          expandAllPipelines: false,
        }
        visible: {
          exportBtn: true,
          labelBtn: true,
          layerBtn: true,
          pipelineBtn: true,
          sidebar: true,        
        }
        theme: "dark",
        tag: {
          enabled: {companies: true}
        },
        nodeType: {
          disabled: {parameters: true}
        }
      }
  />
 </div>
```

Reply:
since we go in details later maybe just 
```
<div style={{height: `100vh`}}>
  <KedroViz
      data={json}
      options={/* Options to configure Kedro Viz */}
  />
</div>
```